SYCL/XPTI/basic_event_collection.cpp

```diff
@@ -3,6 +3,8 @@
 // RUN: %clangxx -fsycl -fsycl-unnamed-lambda -fsycl-targets=%sycl_triple %s -o %t.out
 // RUN: env XPTI_TRACE_ENABLE=1 env XPTI_FRAMEWORK_DISPATCHER=%xptifw_dispatcher env XPTI_SUBSCRIBERS=%t_collector.dll env SYCL_DEVICE_FILTER=opencl %t.out | FileCheck %s 2>&1
 
+// It looks like order of events diffres on Windows
+// XFAIL: windows
```

Comment:
I think writing a single test here gets too complex. We should probably simply have two distinct files for each of OSes, like we do for some other tests.

Reply:
I will update the XPTI related tests. To have them smaller.